src/backend/distributed/planner/multi_logical_optimizer.c

```diff
@@ -54,6 +54,19 @@ int LimitClauseRowFetchCount = -1; /* number of rows to fetch from each task */
 double CountDistinctErrorRate = 0.0; /* precision of count(distinct) approximate */
 
 
+typedef struct MasterAggregateWalkerContext
+{
+	bool repartitionSubquery;
+	AttrNumber columnId;
+} MasterAggregateWalkerContext;
+
+typedef struct WorkerAggregateWalkerContext
+{
+	bool repartitionSubquery;
+	List *expressionList;
+	bool createGroupByClause;
+} WorkerAggregateWalkerContext;
```

Comment:
As far as I remember, we need 2 new lines between the definition of structs and declaration of functions.